templates/email/html/order_successful_self_service.php

```diff
@@ -52,20 +52,16 @@
     <tbody>
 
 
-        <tr><td style="padding-top:20px;">
-
-
-        <?php if (Configure::read('app.sendTaxViaMail')) { ?>
-
+        <?php if (Configure::read('app.showTaxInOrderConfirmationEmail')) { ?>
```

Comment:
I made the bracket after the padding so that in case you don't want to show the tax it has some distance to the table before the next message.